save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/LnkOrganizationTestSuiteDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.permission.Rights
+import com.saveourtool.save.testsuite.TestSuiteDto
+
+/**
+ * @property organization organization that is connected to [testSuite]
+ * @property testSuite manageable test suite
+ * @property rights [Rights] that [organization] has over [testSuite]
+ */
+data class LnkOrganizationTestSuiteDto(
+    var organization: OrganizationDto,
```

Comment:
do we need `var` in data class? and also maybe add `Serializable`

Reply:
No need in var, you are right, will make it to be `val` and add `@Serializable`